vkdgen.py

```diff
@@ -113,7 +113,9 @@ def convertTypeArray(typ, name):
 class DGenerator(OutputGenerator):
 	def __init__(self, errFile=sys.stderr, warnFile=sys.stderr, diagFile=sys.stderr):
 		super().__init__(errFile, warnFile, diagFile)
-	
+		self.enumConstants = []
+		self.funcNames = []
+

```

Comment:
Why is reading the C headers a hassle? The D bindings are almost exactly the same; they have to be, otherwise the bindings would not be ABI compatible. It's trivial to convert the C definitions to D (except for some very rare edge cases like `module` being renamed to `_module`).

I personally prefer to use the specification as the reference, but I understand if you want to use the headers. But the D bindings are hardly any different than the C headers.

If you really want this, then submit a pull request changing it (and with _only that change_), and I will add it if it is implemented well enough.